contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
-    "compiler-version": ["error", "^0.6.12"],
+    "compiler-version": ["warn", "^0.6.12"],
     "const-name-snakecase": "off",
```

Comment:
Finally, attempting to perform a withdrawal on Arbitrum fails with the following JSON RPC error:

<img width="356" alt="Screen Shot 2022-03-17 at 11 27 27 PM" src="https://user-images.githubusercontent.com/1479802/158932387-1787f0f1-1280-4d34-9953-35c4393d1e75.png">

Reply:
I think you forgot to show the JSON RPC error here

Reply:
Ahh, you're right. Sorry about that. Whatever the error was, it didn't occur on my latest test. I was able to do a withdrawal. However, it seems our "sweeping" algorithm needs some work. I had sent 0.01 ETH, and it [left behind](https://arbiscan.io/address/0x2973a807ea703c750fa38e9a43409fbe23ceeb8e) more than half in the stealth address.

Reply:
No RPC errors and Arbitrum withdrawals now leave very little dust 👍